test/builder_test.js

```diff
@@ -255,7 +318,9 @@ describe('Builder', function() {
           builder = new FixtureBuilder(new broccoliSource.UnwatchedDir('test/fixtures/basic'));
 
           expect(builder.watchedPaths).to.deep.equal([]);
-          expect(builder.unwatchedPaths).to.deep.equal(['test/fixtures/basic']);
+          expect(builder.unwatchedPaths.map(paths => paths.path)).to.deep.equal([
```

Comment:
Is this a breaking API change? (It may not matter, but we should consider...)

Reply:
It is yeah, not sure the best way to handle this, the watcher adapter needs access to the node for each watched path...

Reply:
> Is this a breaking API change?

Technically yes, but its not a "public" api breaking change so I don't believe we need to do a major version bump for it.

Reply:
It sure that’s true, Broccoli builder can be used by anything and that property is part of the public interface the watcher adapter uses. Might be worth setting a new property and leave watchedPaths as-is. I wonder if just a sourceNodes property would be good enough with a marker for warched: true and extract the path from the node in the watcher adapter.